mix.exs

```diff
@@ -19,6 +20,12 @@ defmodule ExDoc.Mixfile do
     [clean: [&clean_test_fixtures/1, "clean"]]
   end
 
+  defp package do
+   [licenses: ["Apache 2.0"],
+    links: %{"GitHub" => "https://github.com/elixir-lang/ex_doc",
+             "Contributors" => "https://github.com/elixir-lang/ex_doc/graphs/contributors"}]
```

Comment:
I think contributors must be a list of names. That's why I typically leave it out. :)